zerver/lib/actions.py

```diff
@@ -234,14 +234,14 @@ def send_signup_message(sender, signups_stream, user_profile,
                               realm=user_profile.realm)
 
     internal_send_message(sender,
-            "stream", signups_stream, user_profile.realm.domain,
-            "%s <`%s`> just signed up for Zulip!%s(total: **%i**)" % (
-                user_profile.full_name,
-                user_profile.email,
-                internal_blurb,
-                user_count,
-                )
-            )
+                          "stream", signups_stream, user_profile.realm.domain,
+                          "%s <`%s`> just signed up for Zulip!%s(total: **%i**)" % (
+                              user_profile.full_name,
+                              user_profile.email,
+                              internal_blurb,
+                              user_count,
+                              )
+                          )
```

Comment:
For cases like this where we end up adding a ton of indentation to several, relatively long lines, I think it'd be a better solution to wrap the `sender` variable to the next line, rather than indenting everything so deeply.